index.html

```diff
@@ -7,7 +7,82 @@
     <link rel="stylesheet" href="style.css">
     <title>Assignment 1</title>
 </head>
+
 <body>
-    <h1>comproDLS</h1>
+    <main>
+        <div class="container">
+            <div class="container__left">
+                <div class="info__area">
```

Comment:
double underscore in `info__area` suggests `area` is child of `info`  which is not the case. Similar issues at multiple places.